lib/src/backend/invoker.dart

```diff
@@ -348,6 +352,14 @@ class Invoker {
         await _outstandingCallbacks.noOutstandingCallbacks;
         if (_timeoutTimer != null) _timeoutTimer.cancel();
 
+        if (liveTest.state.result != Result.success &&
+            _runCount < liveTest.test.metadata.retry + 1) {
+          _controller
+              .message(new Message.retry("Retry: ${liveTest.test.name}"));
```

Comment:
Unlike skips, there are valid reasons to mark a test as retried for the long term—for example, that's probably what we'll do for our browser tests, since browsers can sometimes just fail or time out arbitrarily. I don't want to force a super-visible message for those cases.